docs/UserGuide.md

```diff
@@ -403,36 +403,36 @@ For detailed explanations of each flag and acceptable arguments, refer to Sectio
 - **On Success:**
     - Message:
       ```
-      Customer <INDEX> has been updated successfully.
+      client <INDEX> has been updated successfully.
```

Comment:
The message for DeleteCommand has a capitalised 'Client' so I think this should be capitalised as well

Reply:
good catch!

Reply:
Nice catch!